hub/powertoys/run.md

```diff
@@ -104,23 +104,28 @@ The PowerToys Run settings menu includes a plugin manager that allows you to ena
 
 PowerToys Run enables a set of system level actions that can be executed.
 
-| Action command / Search result | Action | Note |
-| :--- | :--- | :--- |
-| `Shutdown` | Shuts down the computer | |
-| `Restart` | Restarts the computer | |
-| `Sign Out` | Signs current user out | |
-| `Lock` | Locks the computer | |
-| `Sleep` | Sleeps the computer | |
-| `Hibernate` | Hibernates the computer | |
-| `Empty Recycle Bin` | Empties the recycle bin | |
-| `UEFI Firmware Settings` | Reboot computer into UEFI Firmware Settings | Only available on systems with UEFI firmware.<br />(Requires administrative permissions.) |
-| `IP address` | Shows the ip addresses from the network connections of your computer. | The search query has to start with `IP` or `address`. |
-| `MAC address` | Shows the mac addresses from the network adapters in your computer. | The search query has to start with `MAC` or `address`. |
+> [!TIP]
+> By default you can search the system commands by their localized name if your system language is supported by PowerToys application.
+> But if you prefere to search them by their english names you can enabled the non-localized search by changing the plugin setting **'Use localized system commands instead of English ones'** to **disabled** in the [plugin manager](#plugin-manager).
 
+| Command / Search result | Localized command name | Action | Note |
+| :--- | :--- | :--- | :--- |
+| `Shutdown` | _Shutdown_ | Shuts down the computer | |
+| `Restart` | _Restart_ | Restarts the computer | |
+| `Sign Out` | _Sign Out_ | Signs current user out | |
+| `Lock` | _Lock_ | Locks the computer | |
+| `Sleep` | _Sleep_ | Sleeps the computer | |
+| `Hibernate` | _Hibernate_ | Hibernates the computer | |
+| `Empty Recycle Bin` | _Empty Recycle Bin_ | Empties the recycle bin | |
+| `UEFI Firmware Settings` | _UEFI Firmware Settings_ | Reboot computer into UEFI Firmware Settings | Only available on systems with UEFI firmware.<br />(Requires administrative permissions.) |
+| `IP address` * | _IP address_ | Shows the ip addresses from the network connections of your computer. | The search query has to start with the word `IP` or the word `address`. |
+| `MAC address` * | _MAC address_ | Shows the mac addresses from the network adapters in your computer. | The search query has to start with the word `MAC` or the word `address`. |
+
+_*) This command may take some time to provide the results._
```

Comment:
```suggestion
_\*) This command may take some time to provide the results._
```

Reply:
why?

Reply:
Small thing, it works now, but idea is to prevent possible issue with formatting.

Reply:
do we need it in the table too?

Reply:
Nah, that's a different thing. It was just to be on the safe side, but it works fine either way